jdaviz/configs/cubeviz/plugins/tests/test_export_plots.py

```diff
@@ -101,3 +103,61 @@ def test_export_plot_exceptions(cubeviz_helper):
 
     with pytest.raises(ValueError, match="Invalid path"):
         plugin.save_figure(filename="/fake/path/image.png")
+
+    with pytest.raises(NotImplementedError, match="save_rgbfits is not available"):
+        plugin._obj.save_rgbfits()
+
+
+# TODO: This will make more sense when we re-organize where plugins go,
+# which would also affect their tests.
+# See https://jira.stsci.edu/browse/JDAT-3200
+
+def test_export_plot_imviz_rgbfits_exceptions(imviz_helper):
+    plugin = imviz_helper.plugins["Export Plot"]
+
+    plugin._obj.movie_enabled = False
+    with pytest.raises(ValueError, match="save_rgbfits is currently disabled"):
```

Comment:
@kecnry , I did add the disable hook here.

Reply:
ah I just missed this because of the name.  Should we call this traitlet something else now that it applies to more than just movies?

Reply:
Since it is not public API and there is a plan to refactor when we use Solara and stuff, I won't bother renaming it.